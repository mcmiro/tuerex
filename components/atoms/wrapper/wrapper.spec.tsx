import * as React from "react";
import {screen, render} from "@testing-library/react";
import Wrapper from "./";

describe("Test Wrapper component", () => {
	it("Should pass when children on ThumbnailDots is exist", () => {
		render(<Wrapper data-testid="wrapper-test">Test</Wrapper>);

		const wrapper = screen.queryByTestId("wrapper-test");
		expect(wrapper).toBeDefined();
		expect(wrapper).toMatchSnapshot();
	});
});
