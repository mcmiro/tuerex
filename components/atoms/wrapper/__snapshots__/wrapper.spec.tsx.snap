// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Wrapper component Should pass when children on ThumbnailDots is exist 1`] = `null`;
